src/index.jsx

```diff
@@ -66,13 +90,21 @@ function configure() {
 apiClient.ensurePublicOrAuthenticationAndCookies(
   window.location.pathname,
   (accessToken) => {
+    callbackAccessToken = accessToken;
+    checkAuthState('ensurePublicOrAuthenticationAndCookies:callback', apiClient.getAuthenticationState());
+
     const { store, history } = configure();
 
     ReactDOM.render(<App store={store} history={history} />, document.getElementById('root'));
 
     if (accessToken) {
       identifyAuthenticatedUser(accessToken.userId);
     } else {
+      // TODO: Could this ever happen?
+      // - If this did happen, wouldn't we want to raise an error?
+      // - Adding error logging for now.
+      NewRelicLoggingService.logError('Empty accessToken returned from ' +
```

Comment:
Can you prefix these log statements so it's clear to developers from which application a log entry originated from?   Or does the `NewRelicLoggingService` already add a prefix?

Reply:
It gets logged with a bunch of meta data in NewRelic Insights, which should include info about the transaction.  Also, I don't think anyone is looking at this particular data, because you'd have to know what you are looking for.